src/test/java/io/aiven/connect/jdbc/sink/JdbcDbWriterTest.java

```diff
@@ -77,6 +77,54 @@ private JdbcDbWriter newWriter(final Map<String, String> props) {
         return new JdbcDbWriter(config, dialect, dbStructure);
     }
 
+    @Test
+    public void shouldGenerateNormalizedTableNameForTopic() {
```

Comment:
Can we blend this function and `shouldGetNormalizedTableName`? They seem to be testing pretty much the same.

Reply:
done